save_coefs.py

```diff
@@ -0,0 +1,52 @@
+from main import DenoisingAutoEncoder
+import numpy as np
+from skimage import measure
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d.art3d import Poly3DCollection
+import torch
+from torch import FloatTensor
+import open3d as o3d
+import os
+import random
+import json
+from tqdm import tqdm
+
+
+M = np.array([[0.80656762, -0.5868724, -0.07091862],
+              [0.3770505, 0.418344, 0.82632997],
+              [-0.45528188, -0.6932309, 0.55870326]])
+
+
+def PointCloud(points, colors=None):
+    pcd = o3d.geometry.PointCloud()
+    pcd.points = o3d.utility.Vector3dVector(points)
+    if colors is not None:
+        pcd.colors = o3d.utility.Vector3dVector(colors)
+    return pcd
+
+
+ckpt = torch.load("epoch=60.ckpt", map_location=torch.device("cuda"))
```

Comment:
Please kindly add configurable ckpt path and whether to use cuda

Reply:
CUDA should be a must for this repo I suppose